package.json

```diff
@@ -96,5 +105,6 @@
         ]
       }
     ]
-  }
+  },
+  "types": "dist/index.d.ts"
```

Comment:
this file doesnt actually exist, also, it seems that the button types have been put in the root. of the dist, you can see that when building or running `npm pack`:

![image](https://user-images.githubusercontent.com/492636/82448770-7c5fea80-9aaa-11ea-8ecb-b7c1f1f1073e.png)

I think these types should be in the actual component folders right? check out how other projects do it

Reply:
it was there when I renamed index.js to ts but finally reverted that at some point.
what should I do? 🤔

Reply:
try and work out what the correct layout should be, I personally don't know. Maybe find a project that has a similar layout (multiple files in dist) and see where the type def files are, and what the types in packagejson is set to

Reply:
I would guess they should be in `dist/button/button.d.ts` for example